src/context/whoami/WhoAmI.tsx

```diff
@@ -4,12 +4,10 @@ import i18n from "../../i18n";
 import type WhoAmIRepresentation from "keycloak-admin/lib/defs/whoAmIRepresentation";
 import type { AccessType } from "keycloak-admin/lib/defs/whoAmIRepresentation";
 import { useAdminClient, useFetch } from "../auth/AdminClient";
+import { homeRealm } from "../../util";
 
 export class WhoAmI {
-  constructor(
-    private homeRealm?: string | undefined,
-    private me?: WhoAmIRepresentation | undefined
-  ) {
+  constructor(private me?: WhoAmIRepresentation | undefined) {
```

Comment:
`me` is already marked as optional, so it's type does not have to be overloaded into `undefined`.
```suggestion
  constructor(private me?: WhoAmIRepresentation) {
```